app/(auth)/signUp/page.tsx

```diff
@@ -0,0 +1,142 @@
+'use client';
+import axios from 'axios';
+import Image from 'next/image';
+import Link from 'next/link';
+import { useForm } from 'react-hook-form';
+import Button from '../../Components/Button/Button';
+import { ButtonTypeEnum } from '../../Components/Button/enums/button-type.enum';
+import Heading from '../../Components/Heading/Heading';
+import { HeadingTypeEnum } from '../../Components/Heading/enums/heading-type.enum';
+import Text from '../../Components/Text/Text';
+import { TextHtmlTypeEnum } from '../../Components/Text/enums/text-html-type.enum';
+import { TextTypeEnum } from '../../Components/Text/enums/text-type.enum';
+import styles from './sign-up.module.scss';
+import { SignUpTypes } from './types/sign-up.type';
+
+const SignUp: SignUpTypes = () => {
```

Comment:
alternative of type is const SignUp: FC

FC means functional component

Reply:
<3 wow